### DISCUSSION TITLE
Release in 6-12 months - App or Pages?

### DISCUSSION THREAD

**Discussion by u840903** [vercel/next.js#47925] [Help] [answered]
### Summary

If I start a new project today that will take between 6 to 12 months to develop. Should I go for pages that is stable now or the app router that is currently in beta?

**Comment by amyegan** [comment, marked as answer]
It's very close! With a timeline of 6-12 months, I would definitely go with `app` rather than `pages`. You can see the features that are already supported in the [App Router Roadmap](https://beta.nextjs.org/docs/app-directory-roadmap).

**Reply by u840903** [reply]
Thank you @amyegan - Very helpful!

**Reply by Adityak4201** [reply]
@amyegan  we are planning to move to production within a month. Should we use app or pages to develop?

**Reply by amyegan** [reply]
I would personally go with `app` at this point since the features that I need are already supported. But the right choice for you depends on the site you're building and the features you need.

**Comment by philwolstenholme** [comment]
The roadmap is almost at 70% (https://next-13-app-roadmap-status.philgw.workers.dev) complete. The big remaining thing is mutations (changing data on the server through form actions, passing functions around etc).

It depends on what type of app you are building. For personal sites, portfolios, etc `app` is probably fine right now.

I'm working on a team releasing an ecommerce site later in the year and we are using `app`.
